### PR TITLE
Include yarn equivalent for existing npm configs and corresponding docs

### PR DESCRIPTION
**Pull request by DaneTheory**
Include yarn equivalent for existing npm configs and corresponding docs

## Checklist

- [ ] Latest code from master has been merged into the pull request branch
- [ ] Honors [the seven code virtues](https://pragprog.com/magazines/2011-08/how-virtuous-is-your-code)
  - [ ] Working, as opposed to incomplete
  - [ ] Unique, as opposed to duplicated
  - [ ] Simple, as opposed to complicated
  - [ ] Clear, as opposed to puzzling
  - [ ] Easy, as opposed to difficult
  - [ ] Developed, as opposed to primitive
  - [ ] Brief, as opposed to chatty
- [ ] Code is camelCased
- [ ] No commented out code (if required, place // TODO above with explanation)
- [ ] No linting issues
- [ ] Automated tests exist and pass
- [ ] Build is successful (`npm run build`)
- [ ] Works in IE 11, Chrome, Firefox, and Edge

## Thanks!

:heart:

### CODE REVIEW

**Comment by coryhouse** [comment]
Hi @DaneTheory - Thanks for the PR! However, my take on Yarn is different. I consider npm the default. If someone prefers to use Yarn instead, I consider it their responsibility to know the corresponding commands. I'd rather not duplicate docs for Yarn. So I'm sorry, but I don't see this as a good fit to merge. But thank you for the thoughtful PR.

**Comment by DaneTheory** [comment]
@coryhouse No worries! I asked about this in a feature request, and seemed like the consensus was that it was good to move forward with. As a side, this PR does improve upon the overall documentation. I can remove the Yarn bits and resubmit without much effort at all. Are you opposed to the additions this PR has to docs?

**Comment by coryhouse** [comment]
I may be open to that but I'm honestly swamped at the moment trying to finish a course update so I'll circle back on this when that's done. Thanks for your patience. :)

**Comment by kwelch** [comment]
I believe this improved the overall docs, but I may need to take a closer look at it once all the yarn bits are removed.

**Comment by DaneTheory** [comment]
@coryhouse @kwelch  Yarn bits have been removed! Apologies for the delay. Work has been crazy.

**Comment by nickytonline** [comment]
@DaneTheory, apologies for the long delay here. I'll take a look at this this week.

**Comment by DaneTheory** [comment]
@nickytonline excellent. Thanks!

File: README.md [hunk 1]
```diff
@@ -4,25 +4,85 @@
...
+
+* *__Only A Single Command Line Interface Necessary__*
+
+  No need to spend time setting up complex development debugging, logging, and notification configurations. The terminal that runs your React Slingshot based project is all you'll need. Trim down on desktop clutter and tooling dependencies. The same CLI running your project doubles as an output console which continously streams relevant development data while you work.
+
+* *__No More "JavaScript Fatigue"__*
+
+  Driven by an active community of contributers, React Slingshot sees regular updates which include the latest, bleeding-edge development enhancements. The project itself is [built atop the most powerful, widely understood community resources available](#technologies) for working with React.
```
Comment (nickytonline) [inline README.md]: `contributers` should be `contributors`

File: README.md [hunk 2]
```diff
@@ -4,25 +4,85 @@
...
+    * Both asynchronous and synchronous functionality as UI/UX actions.
+
+    * **_and much more_**
+
+
+* *__Automated Production Ready Distributions__*
+
+  Ready to show off your finished app to the world? Running a single command:
+
+  - __NPM:__ `npm run build`
```
Comment (nickytonline) [inline README.md]: Since it's only documenting npm, we can probably drop the "npm:"

File: package.json [hunk 3]
```diff
@@ -27,10 +44,9 @@
     "test:cover:CI": "npm run test:CI -- --coverage && cat ./coverage/lcov.info | node_modules/coveralls/bin/coveralls.js",
     "test:watch": "jest --watchAll",
     "open:cover": "npm run test:cover && opn ./coverage/lcov-report/index.html",
-    "analyze-bundle": "babel-node ./tools/analyzeBundle.js"
+    "analyze-bundle": "babel-node ./tools/analyzeBundle.js",
+    "rebuildNodeSass": "npm rebuild node-sass"
```
Comment (nickytonline) [inline package.json]: I don't think this npm command adds much. Where you have it documented, I would probably just write run `npm rebuild node-sass`.

File: README.md [hunk 4]
```diff
@@ -4,25 +4,85 @@
...
...ot Github repository, it comes pre-packaged with the codebase for an example app. If and when you decide you no longer need the example app anymore, and are ready to shave off excess code bloat, run built-in scripts that leave you with just the bare essentials needed for development. The example app itself isn't another basic "Todo List", or any other demo you're used to seeing. React Slingshot purposefully takes covering ReactJS/Redux development fundamentals a step further; Showcasing current best practices in using ReactJS alongside Redux, as well as common ReactJS design patterns like "containers", "smart" and "dumb" components, "contolled" versus "un-controlled" components, etc. No 3rd party libraries that abstract out any of the heavy lifting are used. You get a clear, "vanillaJS" approach thats demonstrates the ReactJS way of doing things. It also serves as mini-presentation for some of the functionality React Slingshot offers for rapid application development. In terms of what you can expect to learn, the example app provides minimal, yet effective, illustrations of a few complex ReactJS/Redux design concepts you'd commonly work to implement in many projects. These include:
```
Comment (nickytonline) [inline README.md]: Not sure about the wording. Some of it sounds sales pitchy a bit. I'll let @coryhouse chime in on this.

**Comment by coryhouse** [comment]
Hi @DaneTheory - Thanks for the PR. However, I'd prefer to close:
1. I'd like to avoid having to maintain docs for both Yarn and npm. Those who choose to use yarn should understand how to translate npm commands.
2. The other significant wording changes are significant verbose rewrites, not grammar fixes. I prefer brevity in docs, again for both the reader and the maintainer.

**Pull request closed without merging**